storage-api/src/test/org/apache/hadoop/hive/ql/exec/vector/expressions/TestStringExpr.java

```diff
@@ -20,6 +20,7 @@
 
 import org.junit.Test;
 
+import java.io.ByteArrayOutputStream;
 import java.nio.charset.StandardCharsets;
 
 import static org.junit.Assert.*;
```

Comment:
Can you help to fix the wildcard * import as well?

Reply:
Got it. I fixed that.